Dynatrace/dynatrace-operator
# Refresh connection info on missing tenant-token secret

# Description

If the tenant token secret is deleted, the operator will ignore the last request timestamp and will recreate the secret and update the connection info.

The dynakube-controller now `Owns` configmap and secrets, so it will react to secret and configmap deletions,

## How can this be tested?
Create a dynakube with oneagent/active-gate configured.
Delete the connection-info configmaps or the tenant token secrets
The deleted secret/configmap should be recreated immediately.

## Checklist
